recipes/coppeliasim-zmqremoteapi-client/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "coppeliasim-zmqremoteapi-client" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/coppeliasim_zmqremoteapi_client-{{ version }}.tar.gz
+  sha256: a97cfe1510e1a41cdabe829efb6e0e0aa58f0612938c839bc3780e0f9c794f7a
+  patches:
+    - fix-cbor2-req.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - pip
+  run:
+    - python >=3.8
+    - pyzmq
+    - cbor2
+
+test:
+  imports:
+    - coppeliasim_zmqremoteapi_client
+  commands:
+    - pip check
+    - python -c 'from coppeliasim_zmqremoteapi_client import RemoteAPIClient; client = RemoteAPIClient()'
```

It is noarch but  this should fix the test in case we need to run this recipe on Windows.

```suggestion
    - python -c "from coppeliasim_zmqremoteapi_client import RemoteAPIClient; client = RemoteAPIClient()"
```